pandas/core/groupby.py

```diff
@@ -233,6 +233,10 @@ class Grouper(object):
 
     >>> df.groupby(Grouper(level='date', freq='60s', axis=1))
     """
+    _attributes = collections.OrderedDict((('key', None), ('level', None),
```

Comment:
as i said before these should just be a list
and _attributes is already defined IIRC you are overwriting it i think